File: FrontPKI/FrontPKI/src/app/DTO/Certificate/CreateIntermediateDTO.ts
export class CreateCertificateDTO{
	issuerId: number = 0;
	cn: String = "";
	organization: String = "";
	organizationUnit: String = "";
	notBefore: String = "";
	notAfter: String = "";
	pathLenConstraint: number = 0;
	isEndEntity: boolean = false;
}

File: PKI/src/main/java/rs/tim33/PKI/DTO/Certificate/CreateIntermediateDTO.java
package rs.tim33.PKI.DTO.Certificate;

import java.time.LocalDateTime;

import com.fasterxml.jackson.annotation.JsonFormat;

public class CreateIntermediateDTO {
	//The certificate
	public Long issuerId;
	public String cn;
	public String organization;
	public String organizationUnit;
	public LocalDateTime notBefore;
	public LocalDateTime notAfter;
	public Integer pathLenConstraint;
}
